docs/tool/GUI.ModsManager/listpatch_information.md

```
@@ -0,0 +1,701 @@
+# [OpenKh Tool Documentation](../index.md) - KH2 Mod Manager
+
+This document contains detailed information on the various listpatches available to use with the Mod Manager.
+
+
```

markdownlint-cli2 (0.22.0)

[warning] No empty links

(MD042, no-empty-links)